test/gravity.db.schema

```diff
@@ -0,0 +1,94 @@
+PRAGMA foreign_keys=OFF;
```

Comment:
We probably want this on.

Reply:
This also comes 1:1 from the `.dump` command. We are not using `foreign` keys anywhere.

Reply:
If we do start using foreign keys, we can enable this at that time.